src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -29,6 +30,9 @@
 
 final class ManagingProductsContext implements Context
 {
+    /** @var ApiClientInterface */
+    private $adminClient;
```

Comment:
Add this as the second argument, because `$client` is the default API client for this context and it should be imo the first one
```suggestion
    private $adminUsersClient;
```
or
```suggestion
    private $administratorsClient;
```